### DISCUSSION TITLE
Which split rule do you use in classif.rpart

### DISCUSSION THREAD

**Discussion by christineaa** [mlr-org/mlr3#730] [Q&A]
Hi,
Which split rule do you use in classif.rpart? Can I choose gini or information gain to be the split rule?

**Comment by pat-s** [comment]
We don't implement custom rules for learners, we just offer what the learners offer in their upstream package.
You can check the hyperparameters in a learners param set.
Check this [section on learners](https://mlr3book.mlr-org.com/basics.html#learner-api) in the mlr3book for more information.
